llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -3087,6 +3087,8 @@ bool SPIRVInstructionSelector::selectIntrinsic(Register ResVReg,
     return selectExtInst(ResVReg, ResType, I, CL::fract, GL::Fract);
   case Intrinsic::spv_normalize:
     return selectExtInst(ResVReg, ResType, I, CL::normalize, GL::Normalize);
+  case Intrinsic::spv_refract:
+    return selectExtInst(ResVReg, ResType, I, GL::Refract);
```

Comment:
if you are using the `GL `selectExtInst` overload then add an OpenCL test to confirm we error see `llvm/test/CodeGen/SPIRV/opencl/reflect-error.ll`